docs/integrations/azure-aks/k8s.mdx

```diff
@@ -66,46 +66,16 @@ Start by creating a new managed Kubernetes cluster in AKS. If you already have o
 
 ## **Step 2**: Install the ngrok Kubernetes Operator {#install-the-ngrok-kubernetes-ingress-controller}
 
-We recommend installing the [ngrok Kubernetes Ingress
-Controller](https://github.com/ngrok/ngrok-operator) before your end-user apps to separate the steps around infrastructure vs. app configuration.
+We recommend reserving a domain for ingress if you don't have one already.
+Navigate to the [**Domains** section](https://dashboard.ngrok.com/domains) of
+the ngrok dashboard and click **Create Domain** or **New Domain**. This will be
+your `NGROK_DOMAIN` for the remainder of this guide.
 
-1. Create an ngrok static subdomain for ingress if you don't have one already. Navigate to the [**Domains**
-   section](https://dashboard.ngrok.com/domains) of the ngrok dashboard and click **Create Domain** or **New
-   Domain**. This static subdomain will be your `NGROK_DOMAIN` for the remainder of this guide.
+Now that you have a Kubernetes cluster running on AKS, you can install
+the ngrok Kubernetes Operator to get traffic to your Kubernetes workloads.
 
-1. Add the ngrok Helm repository if you haven't already.
-
-   ```bash
-   helm repo add ngrok https://charts.ngrok.com
-   ```
-
-1. Set up the `AUTHTOKEN` and `API_KEY` exports, which allows Helm to install the Operator using your ngrok credentials. Find your `AUTHTOKEN` under [**Your Authtoken**](https://dashboard.ngrok.com/get-started/your-authtoken) in the ngrok dashboard.
-
-   To create a new API key, navigate to the [**API** section](https://dashboard.ngrok.com/api) of the ngrok dashboard, click the **New API Key** button, change the description or owner, and click the **Add API Key** button. Copy the API key token shown in the modal window before closing, as the ngrok dashboard will not show you the token again.
-
-   ```bash
-   export NGROK_AUTHTOKEN=[YOUR-AUTHTOKEN]
-   export NGROK_API_KEY=[YOUR-API-KEY]
-   ```
-
-1. Install the ngrok Kubernetes Operator with Helm under a new `ngrok-ingress-controller` namespace.
-
-   ```bash
-   helm install ngrok-ingress-controller ngrok/kubernetes-ingress-controller \
-     --namespace ngrok-ingress-controller \
-     --create-namespace \
-     --set credentials.apiKey=$NGROK_API_KEY \
-     --set credentials.authtoken=$NGROK_AUTHTOKEN
-   ```
-
-1. Verify you have successfully installed the ngrok Kubernetes Operator and that pods are healthy.
-
-   ```bash
-   kubectl get pods -l 'app.kubernetes.io/name=kubernetes-ingress-controller' --namespace ngrok-ingress-controller
-
-   NAME                                                              READY   STATUS    RESTARTS   AGE
-   ngrok-ingress-controller-kubernetes-ingress-controller-man2fg5p   1/1     Running   0          27s
-   ```
+Check out our [Operator installation doc](/docs/k8s/installation/install/) for
+details on how to use Helm to install with your ngrok credentials.
 
 ## **Step 3**: Deploy a demo microservices app {#deploy-a-demo-microservices-app}
```

Comment:
The codeblock  in [this section](https://ngrok-docs-git-k8s-integrations-updates-ngrok-dev.vercel.app/docs/integrations/azure-aks/k8s/#deploy-a-demo-microservices-app) is pretty big. I believe you can add a `collapsible` prop to it to make it collapse.

````
```bash title="example" collapsible=true
code here
```
````